src/mlpack/bindings/python/CMakeLists.txt

```
@@ -15,10 +15,9 @@ macro (post_python_bindings)
         PROPERTY INCLUDE_DIRECTORIES)
     add_custom_target(python_configure
         COMMAND ${CMAKE_COMMAND}
-            -D SETUP_PY_IN=${CMAKE_SOURCE_DIR}/src/mlpack/bindings/python/setup.py.in
-            -D SETUP_PY_OUT=${CMAKE_BINARY_DIR}/src/mlpack/bindings/python/setup.py
+            -D DGENERATE_CPP_IN=${CMAKE_SOURCE_DIR}/src/mlpack/bindings/python/setup.py.in
+            -D DGENERATE_CPP_OUT=${CMAKE_BINARY_DIR}/src/mlpack/bindings/python/setup.py
```

```suggestion
            -D GENERATE_CPP_IN=${CMAKE_SOURCE_DIR}/src/mlpack/bindings/python/setup.py.in
            -D GENERATE_CPP_OUT=${CMAKE_BINARY_DIR}/src/mlpack/bindings/python/setup.py
```
I guess this must be.